src/AuthModule/components/Login/Login.tsx

```diff
@@ -1,11 +1,128 @@
+/* eslint-disable @typescript-eslint/no-unused-vars */
+import { useForm } from "react-hook-form";
+import logo from "../../../assets/images/PMS 3.png";
+import { Link, useNavigate } from "react-router-dom";
+import { FormData } from "../../../interfaces/Auth";
+import { useState } from "react";
+import {useToast} from '../../../context/TostifyContext'
+import axios from "axios";
 
+export default function Login({savLoginData}) {
+  // All states here on the top 
+  const [showPassword, setShowPassword] = useState<boolean>(false);
+  const { showSuccessToast, showErrorToast } = useToast();
+  // note we will move it to context for using 
+  const [spinner, setSpinner] = useState<boolean>(false);
+  const {
+    register,
+    handleSubmit,
+    formState: { errors },
+  } = useForm<FormData>();
+  const navigate = useNavigate();
+// test done 
+// Function for password visibility
+//  void indicates that a function does not return any value
+const togglePasswordVisibility = (): void => {
+  setShowPassword((prevState: boolean) => !prevState);
+};
+  
+// senD Data to Api
+const onSubmit = async (data: FormData) => {
+  setSpinner(true);
+
+  try {
+    const response = await axios.post('https://upskilling-egypt.com:3003/api/v1/Users/Login', data );
+  
+    showSuccessToast('Login successfully');
+    localStorage.setItem("token",response.data.token);
+    savLoginData();
+    navigate('/dashboard');
+    // console.log(response)
+  } catch (error ) {
+    showErrorToast("An error occurred with login..");
+  } finally {
+    setSpinner(false);
+  }
+};
 
-export default function Login() {
   return (
-    <div className="Auth-container vh-100  d-flex align-items-center justify-content-center">
-      <div>
-        <h2 className="text-white ">Loginnnnn</h2>
+  
```

Comment:
The login form's background is different from the one in the FIgma file. I also suggest reducing the padding for the error messages, as it takes too much space and it is currently bigger than the input itself.
![image](https://github.com/Elias-lap/ProjectMangement2/assets/64327685/f1bc25d0-8531-4ac6-9893-ed1d8152d692)

We also need to separate the words in forgetPassword.

Reply:
Also I noticed that the Figma design is using a border-bottom input. We can update the styles for the inputs to match the design. As this skill is crucial  and it indicates that you have a good eye for details and can convert any design to pixel-perfect websites.